store/sqlstore/license_store.go

```diff
@@ -29,6 +31,12 @@ func NewSqlLicenseStore(sqlStore SqlStore) store.LicenseStore {
 func (ls SqlLicenseStore) CreateIndexesIfNotExists() {
 }
 
+// Save validates and stores the license instance in the database. The Id
```

Comment:
```suggestion
// Saves, validates, and stores the license instance in the database. The Id
```

Reply:
Actually what I'm saying here is `Save` as function, "validates and stores the license instance"

Reply:
Ah, got it. You can disregard this change then. :)